src/runtime/StaticStrings.cpp

```diff
@@ -136,8 +137,25 @@ void StaticStrings::initStaticStrings(AtomicStringMap* atomicStringMap)
         buf[1] = 0;
         asciiTable[i].initStaticString(atomicStringMap, new Latin1String(buf, 1));
     }
+
+#define DECLARE_LAZY_STATIC_STRING(Name, unused) m_lazy##Name = AtomicString();
+    FOR_EACH_LAZY_STATIC_STRING(DECLARE_LAZY_STATIC_STRING);
+    FOR_EACH_LAZY_INTL_STATIC_STRING(DECLARE_LAZY_STATIC_STRING);
+#undef DECLARE_LAZY_STATIC_STRING
 }
 
+#define DECLARE_LAZY_STATIC_STRING(Name, stringContent)                                                                           \
+    AtomicString StaticStrings::lazy##Name()                                                                                      \
+    {                                                                                                                             \
+        if (m_lazy##Name.string() == String::emptyString) {                                                                       \
```

Comment:
How about adding `UNLIKELY` macro at this point?

Reply:
Good idea!

Reply:
I added it